Mathlib/Analysis/Distribution/TemperateGrowth.lean

```diff
@@ -174,6 +175,14 @@ theorem _root_.ContinuousLinearMap.bilinear_hasTemperateGrowth [NormedSpace 𝕜
   refine .const_mul_left (.mul (h1 i ?_).norm_left (h2 (n-i) ?_).norm_left) _ <;>
   grind
 
+@[fun_prop]
+lemma _root_.Function.HasTemperateGrowth.id : Function.HasTemperateGrowth (fun (x : E) ↦ x) := by
```

Comment:
Normally the `.id` spelling gets an actual `_root_.id` expression, and the `id'` spelling (or actually, I think we've decided on `fun_id`, but this isn't currently used anywhere in Mathlib I don't think) is reserved for `fun x ↦ x`. Do you want to add that additional spelling?